sale_coupon_multi_gift/security/ir.model.access.csv

```diff
@@ -0,0 +1,3 @@
+id,name,model_id/id,group_id/id,perm_read,perm_write,perm_create,perm_unlink
+access_multi_gift_salesman,salesman,model_sale_coupon_multi_gift,sales_team.group_sale_salesman,1,0,0,0
+access_multi_gift_manager,multi_gift manager,model_sale_coupon_multi_gift,sales_team.group_sale_manager,1,1,1,1
```

Comment:
Is this in line with main coupon tables permissions?

Reply:
Yes